### PR TITLE
Clarification on how to use custom image

### PR DESCRIPTION
**Pull request by maovidal**
This may help a bit to clarify what additional changes are expected when using custom images.
This was taken after the post https://github.com/frappe/frappe_docker/issues/876#issuecomment-1261834758

### CODE REVIEW

**Comment by sdfateh** [comment]
You Just add more lines to the files with the same effect.
Why do you think we needed more sections?

**Comment by revant** [comment]
If you change the main compose.yaml also make sure that the new `easy-install.py` script from frappe/bench works.

I'd think of something that automates `easy-install.py` either using `build:` or replacing `image:`

**Comment by revant** [comment]
Atleast fix the lint. I cannot co-commit on your main branch.

**Comment by maovidal** [comment]
> You Just add more lines to the files with the same effect.
> Why do you think we needed more sections?

Besides identifying that the custom images are only for the workers, scheduler, frontend, backend and not websocket/socketio, I think the proposed modifications follow the use of [extension fragments](https://docs.docker.com/compose/compose-file/#using-extensions-as-fragments), which is already used partially on the current compose files. Not only those may be easier to read, but also may help to maintain and extend that kind of components.

**Comment by maovidal** [comment]
> Atleast fix the lint. I cannot co-commit on your main branch.

I'm sorry about that. I did not checked the CONTRIBUTING guide prior my pull request and I haven't had any experience with any of those tools before. But I have already fixed them and performed the tests locally.

What should I do now? Should I commit those changes?

**Comment by revant** [comment]
yes commit the additional changes with commit message `chore: fix lint`

File: compose.yaml [hunk 1]
```diff
@@ -56,7 +62,7 @@ services:
 
   websocket:
     <<: *depends_on_configurator
-    image: frappe/erpnext:${ERPNEXT_VERSION}
+    image: frappe/erpnext:${ERPNEXT_VERSION:?No ERPNext version set}
```
Comment (revant) [inline compose.yaml:65]: websocket image can also be custom. it'll have Frappe and socketio

**Comment by revant** [comment]
if you pull the latest changes it'll fix the v14 tests

File: compose.yaml (hunk 1 above)
Comment (revant) [reply compose.yaml:65]: @maovidal can you also change this image? No need to keep it. Custom image will have socketio in it.

Comment (maovidal) [reply compose.yaml]: Done.
Hey @revant, maybe not the place nor the time, but certainly I want to express my gratitude to you for all you have shared on this repo. I've learned a lot.

**Pull request merged**